Services/SciMaterials.Contracts/SciMaterials.Contracts.csproj

```
@@ -15,4 +15,10 @@
     </NoWarn>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Reference Include="Microsoft.Extensions.Identity.Stores">
+      <HintPath>..\..\..\..\..\..\usr\local\share\dotnet\shared\Microsoft.AspNetCore.App\6.0.10\Microsoft.Extensions.Identity.Stores.dll</HintPath>
+    </Reference>
+  </ItemGroup>
```

Вот эта проблема давала ошибки
```suggestion
  <ItemGroup>
    <PackageReference Include="Microsoft.Extensions.Identity.Stores" Version="6.0.10" />
  </ItemGroup>
```